### ISSUE TITLE
Paused Episode Automatically Gets Archived

### ISSUE THREAD

**Issue by mchowning** [Automattic/pocket-casts-android#153]
User reports that paused episodes will automatically get archived, especially when they switch to another app with the Pocket Casts app running in the background.

- internal Slack discussion: p1651853290716699-slack-C02A333D8LQ

---

- Most recently observed with the "No Such Thing As Lord Cauldronhead" episode (podcast 308313, episode 642117214).
- Commonly occurs with [No Such Thing As A Fish (podcast 308313) episodes
- We can't locate any event of the above episode getting archived in the debug log
- Regina noticed the user mentioned the downloaded episodes were saved in the SD cards. However, issue occurs no matter what the storage setting is (Phone or SD card)
- Issue occurs even with wired headset
- Not using any other devices or integrations
- We've asked the user to join the Private Alpha

---

Latest findings from the user (quoting):

**tl;dr:** This issue _may_ be occurring with episodes that have been played before, archived, and subsequently un-archived and replayed.  Could this be an issue with how the account marks episodes as played or unplayed?  Or something like that?

**Long-form details:**

I mentioned, in a previous email, that this issue often occurs with the podcast _No Such Thing As a Fish_.  This might be significant, because I am listening to this podcast for _the second time_.  In fact, as the issue intermittently occurs, I've noticed that it seems to happen with episodes I'm re-playing (having played them before some time in the past).  I can't prove this (because I didn't take good notes at first) but this issue might be occurring _exclusively_ with episodes I'm playing for a second time.

1\. Before App Reinstallation

When I first noticed this issue, I often listened to episodes of podcasts that I had fully played before in Pocket Casts.  That is to say that:

> a) the app had played the episode,
> 
> b) marked it played, and
> 
> c) "auto-archived" it. 

When I re-listened to these episodes (months or years later), I did so by

> a) pressing the "Show archived" button in the podcast page in the app,
> 
> b) selecting an episode,
> 
> c) adding it to my "Up Next" list and,
> 
> d) letting it play in queue.

As noted in a previous email, this would intermittently lead to this issue: sometimes the app would mark these episodes as 'archived' in the middle of listening (while the app was backgrounded). 

As previously noted, I tried to resolve this issue by deleting all the Pocket Casts app-related data on my phone, then uninstalling and reinstalling the app. 

2\. After App Reinstallation

After I reinstalled, the app downloaded and displayed a list of all the podcasts to which I was subscribed.  However, every episode in each podcasts was shown as 'unplayed'.  Examples: _No Such Thing As a Fish_, _Reply All_, and _Slow Burn_ had all the episodes marked 'unplayed', even though I had listened to many of the episodes already. 

Some of these episodes I manually re-archived.  I 'left' some other episodes unarchived, anticipating that I would like to listen to them again. 

Subsequent to that I have re-listened to many of these episodes: both the ones I 'left' unarchived, and those I manually re-archived.  The issue seems to occur with episodes in both of these groups.

3.  Is this significant?

This led me to wonder: could there be some significance to the status of an episode as having previously been marked 'archived' or 'played'?  I can imagine a situation in which the app checks back with my account, realises that the episode 'should' be archived, and behaves unexpectedly.

Reported in 5084177-zen
